src/CodeXMLParse.cpp

```diff
@@ -162,11 +162,30 @@ void AnnotateConstantVariable(Varnode *varnode, std::vector<RCodeAnnotation> *ou
 	out->push_back(annotation);
 }
 
+// Annotates local variables and function parameters
+void AnnotateLocalVariable(pugi::xml_node node, std::vector<RCodeAnnotation> *out)
+{
+	pugi::xml_attribute attr = node.attribute("color");
+	if (attr.empty())
+		return;
+	std::string color = attr.as_string();
+	if (color == "")
+		return;
+	
+	RCodeAnnotation annotation = {};
+	annotation.variable.name = strdup(node.child_value());
+	if (color == "param")
+		annotation.type = R_CODE_ANNOTATION_TYPE_FUNCTION_PARAMETER;
+    else if (color == "var")
```

Comment:
```suggestion
	else if (color == "var")
```

Reply:
@karliss  @thestr4ng3r I had seen the decision that leads to color. For PR #124 , I have used info from here. https://github.com/thestr4ng3r/ghidra/blob/6c10f36f06468f866188cccf960c019779fb9028/Ghidra/Features/Decompiler/src/decompile/cpp/variable.cc#L452-L470

In there also, the same logic (if I didn't miss out anything) is used there to classify local variables and parameters. This is the reason why I used color to classify a variable as a parameter and local variable. Have you seen examples where color is mistakenly annotated?